pywbem_mock/_mainprovider.py

```diff
@@ -678,12 +667,12 @@ def get_qualifier_store(self, namespace):
 
     def validate_namespace(self, namespace):
         """
-        Validates that a namespace is defined in the repository.
+        Validates that a namespace is defined in the datastore.
 
           Parameters:
 
             namespace (:term:`string`):
-              The name of the CIM namespace in the mock repository. Must not be
+              The name of the CIM namespace in the datastore. Must not be
               `None`.
 
           Returns:
```

Comment:
This method returns either None or raises an exception. It never returns True. Remove the entire "Returns:" section.

Reply:
DONE